templates/app/views/products/index.html.erb

```diff
@@ -9,9 +9,9 @@
   </aside>
   <section class="products_section col-span-full md:col-span-9 lg:col-span-10">
     <% if params[:keywords] && @products.empty? %>
-      <h6>
+      <span style="text-red drop-shadow-sm text-h6 font-serif md:text-h5 lg:text-h3">
```

Comment:
here we seem to have personal preference as well. I would like to keep these changes solely related to the goal of this PR.

Reply:
In my personal opinion The secondary headings should be pulled from the resource description and not from arbitrarily placed tags. Is there any particular reason you want to keep this a heading other than style?

Reply:
I can restore the styling on this section. I felt a contrast would be better but we can keep it as close as possible to the default.